### PR TITLE
ci: DLT-1602 release notification

### PR DESCRIPTION
**Pull request by juliodialpad**
# Release notification

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will not increment the version number, but will still deploy to documentation site on release:

- [x] CI

## :book: Jira Ticket

https://dialpad.atlassian.net/browse/DLT-1602

## :book: Description

- Added an action to send a message to dialtone channel after a dialtone release is done.

## :bulb: Context

We're missing the notifications messages on release that we had on Slack previously, this will get such messages back.

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.

## :crystal_ball: Next Steps

- Test doing a release

### CODE REVIEW

File: .github/workflows/release-notification.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+jobs:
+  send-sms:
+    if: ${{ startsWith('dialtone/v', env.RELEASE_NAME) }}
+    name: Send SMS
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+    env:
+      API_URL: https://us-central1-dp-dialtone-design-system.cloudfunctions.net/send-sms-to-dialtone-test-channel
```
Comment (braddialpad) [inline .github/workflows/release-notification.yml:19]: is this supposed to be locked to a location like this "us-central1"? No location agnostic url? If expected it's fine.

Comment (juliodialpad) [reply .github/workflows/release-notification.yml:19]: Yes, it's expected. It is something specific of cloud functions version 2

Comment (braddialpad) [reply .github/workflows/release-notification.yml:19]: k np looks good then

**Review by braddialpad — APPROVED** [review]
Excited to see this work!

**Comment by juliodialpad** [comment]
I'll need to change the URL after testing that it is working as expected.

But I need to merge this to be able to test it.

Thanks!

**Pull request merged**